test/unit/validator.test.js

```diff
@@ -391,21 +391,21 @@ describe('The Validation option', function () {
             { detailedBlobValidation: true }),
           historyRequest = [],
           resultObj,
-          violatedKeywords = [
-            'data.items.minProperties',
-            'data.items.required',
-            'data.items.properties.entityId.maxLength',
-            'data.items.properties.accountNumber.minLength',
-            'data.items.properties.entityName.format',
-            'data.items.properties.incType.enum',
-            'data.items.properties.companyNumber.exclusiveMinimum',
-            'data.items.properties.website.type',
-            'data.items.properties.turnover.multipleOf',
-            'data.items.properties.description.pattern',
-            'data.items.properties.wants.uniqueItems',
-            'meta.maxProperties',
-            'meta.additionalProperties'
-          ];
+          violatedKeywords = {
+            'data.items.minProperties': '$.request.body.data[0]',
+            'data.items.required': '$.request.body.data[0]',
+            'data.items.properties.entityId.maxLength': '$.request.body.data[0].entityId',
+            'data.items.properties.accountNumber.minLength': '$.request.body.data[0].accountNumber',
+            'data.items.properties.entityName.format': '$.request.body.data[0].entityName',
+            'data.items.properties.incType.enum': '$.request.body.data[0].incType',
+            'data.items.properties.companyNumber.exclusiveMinimum': '$.request.body.data[0].companyNumber',
+            'data.items.properties.website.type': '$.request.body.data[0].website',
+            'data.items.properties.turnover.multipleOf': '$.request.body.data[0].turnover',
+            'data.items.properties.description.pattern': '$.request.body.data[0].description',
+            'data.items.properties.wants.uniqueItems': '$.request.body.data[0].wants',
```

Comment:
@LuisTejedaS Can we add tests that checks deeeper properties? i.e. `$.request.body.data[0].wants.hello.world`